### PR TITLE
Add note about contributions during refactor period

### PR DESCRIPTION
**Pull request by Darylgolden**
## Overview: What does this pull request change?
Updated the contribution guidelines and README to explain the current plan with how contributions will be handled during the refactor period.

## Reviewer Checklist
- [ ] The PR title is descriptive enough for the changelog, and the PR is labeled correctly
- [ ] If applicable: newly added non-private functions and classes have a docstring including a short summary and a PARAMETERS section
- [ ] If applicable: newly added functions and classes are tested

### CODE REVIEW

File: docs/source/contributing.rst [hunk 1]
```diff
@@ -2,6 +2,13 @@
 Contributing
 ############
 
+.. important:: Manim is currently undergoing a major refactor. In general,
+   contributions implementing new features will not be accepted in this period.
+   Other contributions unrelated to cleaning up the codebase may also take a longer
+   period of time to be reviewed. This guide may quickly become out of data; we
```
Comment (Darylgolden) [inline docs/source/contributing.rst:8]:
```suggestion
   period of time to be reviewed. This guide may quickly become out of date; we
```

File: README.md [hunk 2]
```diff
@@ -112,6 +112,12 @@ Server](https://www.manim.community/discord/) or [Reddit Community](https://www.
 
 Contributions to Manim are always welcome. In particular, there is a dire need for tests and documentation. For contribution guidelines, please see the [documentation](https://docs.manim.community/en/stable/contributing.html).
 
+However, please note that Manim is currently undergoing a major refactor. In general,
+contributions implementing new features will not be accepted in this period.
+The contribution guide may become out of date; we highly recommend joining our
```
Comment (Darylgolden) [inline README.md:117]:
```suggestion
The contribution guide may become outdated quickly; we highly recommend joining our
```

File: docs/source/contributing.rst [hunk 3]
```diff
@@ -2,6 +2,13 @@
 Contributing
 ############
 
+.. important:: Manim is currently undergoing a major refactor. In general,
+   contributions implementing new features will not be accepted in this period.
+   Other contributions unrelated to cleaning up the codebase may also take a longer
+   period of time to be reviewed. This guide may quickly become out of date; we
```
Comment (Darylgolden) [inline docs/source/contributing.rst:8]:
```suggestion
   period of time to be reviewed. This guide may quickly become outdated quickly; we
```

**Review by behackl — APPROVED** [review]
SGTM :-)

**Pull request merged**